styles/main.css

```diff
@@ -95,19 +110,43 @@ section, footer, div {
     height: 60px;
     text-align: center;
     line-height: 1;
-    padding: 10px 30px;
+    padding: 24px 35px;
     margin-top: -1px;
     font-weight: 600;
     letter-spacing: .1rem;
     transition: all 0.2s linear;
 }
 
+@media (max-width: 550px) {
+    
+    #footer-email #email {
+        display: grid;
+        margin-bottom: -60px;
+        grid-template-columns: 10px 1fr 10px;
+        grid-template-rows: repeat(2, auto);
+    }
+    
+    #footer-email #email #email-box {
+        grid-column: 2/3;
+        width: auto;
+        margin-left: 30px;
+        margin-right: 30px;
+    }
+    
+    #footer-email #email #email-btn {
+        grid-column: 2/3;
+        margin-top: 20px;
+        justify-self: center;
```

Comment:
This isn't a valid property! The button looks fine though. I guess that this can just be removed

Reply:
What is the invalid property ?! all what you see there is valid, and if you removed it, it will break the button

Reply:
Oops! I'm very sorry I did that! It's the `justify-self` property. I ran it through the W3C's [CSS Validator](http://jigsaw.w3.org/css-validator/) and it wasn't recognised because it's in the Working Draft. I see now that it affects the button in a narrow viewport. I shall re add it. I apologise again. I didn't take enough care over this.

Reply:
haha no problem :) actually it's my fault I should have said that I used CSS grid (was just exploring it this morning...very powerful by the way)